community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,460 @@
+            },
+            "openshift": true
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaServer",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "add": false,
+            "serviceAccount": "aqua-sa",
+            "dbSecretName": "aqua-database-password",
+            "dbSecretKey": "db-password",
+            "rbac": {
+              "enable": true,
+              "openshift": true,
+              "privileged": true
+            },
+            "deploy": {
+              "replicas": 1,
+              "service": "LoadBalancer",
+              "image": {
+                "repository": "console",
+                "registry": "registry.aquasec.com",
+                "tag": "3.5",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "aquaDb": "aqua-database",
+            "adminPassword": "Password1",
+            "licenseToken": null,
+            "clusterMode": true,
+            "ssl": false,
+            "auditSsl": false,
+            "dockerless": false,
+            "openshift": true
+          }
+        }
+      ]
+spec:
+  displayName: Aqua Security Operator
+  version: 0.0.1
+  description: |-
+    The Aqua Security Operator runs within a Kubernetes or Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components as
+    * Server (Console)
+    * Gateway
+    * Database (Not recommended for production environments)
+    * Enforcer (Agent)
+    * Scanner CLI
+    * CSP (Package of Server, Gateway and Database)
+
+    Use the aqua-operator to - 
+    * Deploy Aqua Security components on Kubernetes or Openshift
+    * Scale up Aqua Security components with extra replicas
+    * Assign metadata tags to Aqua Security components
+    * Automatic scale to the Aqua Scanner CLI by the count of the Scanning Queue.
+
+    ## Before You Begin
+    ### Openshift 3.11+
+    you need to create - 
+      * Service account with pull permissions from aqua docker registry
+      * Secret for docker registry 
+      * Secret for database
+
+    ```
+    oc new-project aqua
+
+    oc create serviceaccount aqua-sa -n aqua
+
+    oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:aqua:aqua-sa -n aqua
+    oc adm policy add-scc-to-user privileged system:serviceaccount:aqua:aqua-sa -n aqua
+    oc adm policy add-scc-to-user hostaccess system:serviceaccount:aqua:aqua-sa -n aqua
+
+    oc create secret docker-registry aqua-registry-secret --docker-server=registry.aquasec.com --docker-username=<user name> --docker-password=<password> --docker-email=<user email> -n aqua
+
+    oc create secret generic aqua-database-password --from-literal=db-password=<password> -n aqua
+    oc secrets add aqua-sa aqua-registry-secret --for=pull -n aqua
+    ```
+    ### Kubernetes 1.11+
```

Comment:
You can have different descriptions for each CSV to match the target platform

Reply:
OK

Reply:
This should be the OpenShift version for the community-operators directory and the Kubernetes version for upstream-community-operators directory. Right now it's backwards.

Reply:
I allready update it!